packages/default/scss/dock-manager/_layout.scss

```diff
@@ -0,0 +1,231 @@
+@mixin kendo-dock-manager--layout-base() {
+
+    // DockManager
+    .k-dock-manager {
+        display: flex;
+        flex-flow: row nowrap;
+        position: relative;
+        border-width: $kendo-dock-manager-border-width;
+        border-style: $kendo-dock-manager-border-style;
+
+        .k-pane,
+        .k-splitbar {
+            box-sizing: border-box;
+        }
+
+        .k-pane-flex {
+            flex-direction: column;
+        }
+
+        .k-splitbar-horizontal {
+            border-inline-width: $kendo-dock-manager-splitbar-border-width;
+            border-inline-style: $kendo-dock-manager-splitbar-border-style;
+        }
+
+        .k-splitbar-vertical {
+            border-block-width: $kendo-dock-manager-splitbar-border-width;
+            border-block-style: $kendo-dock-manager-splitbar-border-style;
+        }
+
+        .k-tabstrip-content,
+        .k-splitter {
+            border-width: 0;
+        }
+
+        .k-toolbar .k-separator-horizontal {
+            width:  $kendo-line-height-em;
+            height: 0;
+            border-width: $kendo-dock-manager-separator-border-width 0 0;
+        }
```

Comment:
Actually they are needed because [here](https://github.com/telerik/kendo-themes/blob/develop/packages/default/scss/toolbar/_layout.scss#L81) we explicitly state that the separator in the Toolbar should be vertical